satellite/console/consolesim/mailsender.go

```diff
@@ -0,0 +1,84 @@
+package consolesim
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"strings"
+
+	"github.com/skyrings/skyring-common/tools/uuid"
+
+	"storj.io/storj/internal/post"
+	"storj.io/storj/satellite/console"
+)
+
+// MailsSender is smtp sender mock that ignores sending process
+type MailsSender struct {
+	DB console.DB
+}
+
+// FromAddress return empty mail address
+func (sender *MailsSender) FromAddress() post.Address {
+	return post.Address{}
+}
+
+// SendEmail tries to activate account from email
+func (sender *MailsSender) SendEmail(msg *post.Message) error {
+	if msg.Subject != console.ActivationSubject {
+		return nil
+	}
+
+	content := msg.Parts[0].Content
+	index := strings.Index(content, "Verify your account")
+
+	linkEndOffset := 0
```

Comment:
You can use a regexp here.

Reply:
Is it more efficient?

Reply:
Not really, but it's probably easier to read what's happening and less assumptions how the email is structured.